_source/logzio_collections/_p8s-sources/prometheus-telegraf-output.md

```diff
@@ -0,0 +1,70 @@
+---
+title: Ship Prometheus metrics with the Telegraf collector
+logo:
+  logofile: mascot-telegraf.png #telegraf-tiger.png  #telegraf-logo-preview.svg
+  orientation: vertical
+data-source: Telegraf Collector for Prometheus metrics
+open-source:
+  - title: Telegraf Collector for Prometheus metrics
+    github-repo: logz-telegraf-metrics
+templates: ["docker"]
+contributors:
+  - fadi-khatib
+  - yberlinger
+shipping-tags:  # add tag attribute definition to .yaml for shipping
+  - telegraf
+
+---
+
+
+{% include page-info/early-access.md type="beta" %}
+
+This project allows you to collect Prometheus-format metrics with Telegraf, and ship them to Logz.io, also via Telegraf.
```

Comment:
@YajB, I think we want to keep it as before as we are not configuring Telegraf to collect metrics.
This doc shows how to configure Telegraph in order to send the collected metrics to logz.io